# Parameters for Oskar III.

# Serial connection
serial:
  baudrate: 115200

# Physical properties

phys:
  wheel_diam_m: 0.150
  wheel_dist_m: 0.600 
  wheel_gear_ratio: 6
